topostats/filters.py

```diff
@@ -373,19 +461,21 @@ def average_background(self, image: np.ndarray, mask: np.ndarray = None) -> np.n
         LOGGER.info(f"[{self.filename}] : Zero averaging background : {mean} nm")
         return image - mean
 
-    def gaussian_filter(self, image: np.ndarray, **kwargs) -> np.array:
-        """Apply Gaussian filter to an image.
+    def gaussian_filter(self, image: npt.NDArray, **kwargs) -> npt.NDArray:
+        """
+        Apply Gaussian filter to an image.
 
         Parameters
         ----------
-        image: np.array
-            Numpy array representing image.
+        image : npt.NDArray
+            Numpy array representing the image.
+        **kwargs
+            Keyword arguments passed on to the skimage.filters.gaussian() function.
 
         Returns
         -------
-        np.array
-            Numpy array of gaussian blurred image.
-
+        npt.NDArray
+            Keyword arguments passed on to the skimage.filters.gaussian() function.
```

Comment:
This doesn't seem to describe the correct return value, as I think it has been copied from the **kwargs definition. This should instead be a numpy array that represents the image following Gaussian filtering.